ReleaseNotes.md

```diff
@@ -1,5 +1,54 @@
 # Release Notes
 
+## New in Release 2.5.0
+Post-training Quantization:
+- Features:
+  - Added Post-training Quantization support for OpenVINO IR (OpenVINO backend).
+  - Added `"overflow_fix"` parameter (for `quantize(...)` & `quantize_with_accuracy_control(...)` methods) support & functionality. It improves accuracy for optimized model for affected devices.
+  - Added `"model_type"` parameter (for `quantize(...)` & `quantize_with_accuracy_control(...)` methods) support that allows reach better accuracy for specific model architectures (e.g. transformers).
+  - Added support for in-place statistics collection (reduce memory footprint during optimization).
+  - Added `nncf.parameters.IgnoredScope` parameter (for `quantize(...)` & `quantize_with_accuracy_control(...)` methods) support that allows exclude layers from optimization scope in different ways.
+  - (OpenVINO) Added Quantization with accuracy control algorithm.
```

Comment:
add (`quantize_with_accuracy_control(...)` method)

Reply:
quantize_with_accuracy_control was introduced in NNCF 2.4.0. In this release the POT implementation was ported to NNCF. https://github.com/openvinotoolkit/nncf/blob/9da39ffdb911951bea388c49f85bba3e0d5a3e58/nncf/quantization/algorithms/accuracy_control/algorithm.py#L96

Reply:
> quantize_with_accuracy_control was introduced in NNCF 2.4.0. In this release the POT implementation was ported to NNCF.
> 
> https://github.com/openvinotoolkit/nncf/blob/9da39ffdb911951bea388c49f85bba3e0d5a3e58/nncf/quantization/algorithms/accuracy_control/algorithm.py#L96

ok, then we could skip it.